rpc/src/main/java/com/distkv/dst/rpc/protobuf/sorted_list_pb.proto

```diff
@@ -61,11 +61,23 @@ message RemoveMemberResponse {
     required Status status = 1;
 }
 
+message GetMemberRequest {
+    required string key = 1;
+    required string member = 2;
+}
+
+message GetMemberResponse {
+    required Status status = 1;
+    required SortedListEntity entity = 2;
+    required uint32 count = 3;
```

Comment:
count --> rank （or  something）

Reply:
To unify the below code. I think it doesn't matter.